src/JuMP.jl

```diff
@@ -579,17 +582,70 @@ function getNumSOCRows(m::Model)
     return numSOCRows
 end
 
-function getdual(c::ConstraintRef{Model,SOCConstraint})
-    numBndRows = getNumBndRows(c.m)
-    numSOCRows = getNumSOCRows(c.m)
-    if length(c.m.conicconstrDuals) != (MathProgBase.numlinconstr(c.m) + numBndRows + numSOCRows)
-        error("Dual solution not available. Check that the model was properly solved and no integer variables are present.")
+# Returns the number of semidefinite constraints
```

Comment:
Small terminology point, but I'd think of ``X \in SDP`` as a single semidefinite constraint. SDP rows is okay, just responding to the comment.